features/cellular/easy_cellular/EasyCellularConnection.cpp

```diff
@@ -281,20 +305,28 @@ void EasyCellularConnection::attach(mbed::Callback<void(nsapi_event_t, intptr_t)
 
 void EasyCellularConnection::modem_debug_on(bool on)
 {
-    CellularDevice *dev = _cellularConnectionFSM.get_device();
-    if (dev) {
-        dev->modem_debug_on(on);
+    if (_cellularConnectionFSM) {
+        CellularDevice *dev = _cellularConnectionFSM->get_device();
+        if (dev) {
+            dev->modem_debug_on(on);
+        }
     }
 }
 
 void EasyCellularConnection::set_plmn(const char* plmn)
 {
-    _cellularConnectionFSM.set_plmn(plmn);
+    if (_cellularConnectionFSM) {
```

Comment:
should we take plmn in temporary store or init FSM similar to set_sim_pin

Reply:
Good point, will add init